graylog2-web-interface/src/util/PaginationHelper.js

```diff
@@ -0,0 +1,12 @@
+// @flow strict
+const PaginationHelper = {
+  urlGenerator: (destUrl: string, page: number, perPage: number, query: string, resolve: boolean = true): string => {
+    const url = `${destUrl}?page=${page}&per_page=${perPage}&resolve=${resolve.toString()}`;
```

Comment:
Instead of manual string handcrafting, we could use the [URI.js](https://medialize.github.io/URI.js/) builder to help with escaping et al.